lib/sacastats_web/controllers/character_controller.ex

```diff
@@ -0,0 +1,66 @@
+defmodule SacaStatsWeb.CharacterController do
+  use SacaStatsWeb, :controller
+  alias SacaStatsWeb.CharacterView
+
+  def character(conn, %{"character_name" => name, "stat_type" => stat_template_name}) do
+    # Fetch character info from API and DB,
+
+    if String.downcase(stat_template_name) == "lookup" do
+      redirect(conn, to: Routes.character_path(conn, :character, name, "general"))
+    end
```

Comment:
Instead of using an if statement, we can use another function head + pattern matching.
```suggestion

  def character(conn, %{"character_name" => name, "stat_type" => "lookup"}) do
    redirect(conn, to: Routes.character_path(conn, :character, name, "general"))
  end
  
  def character(conn, %{"character_name" => name, "stat_type" => stat_template_name}) do
```